snmpy

```diff
@@ -188,6 +189,7 @@ if __name__ == '__main__':
 
     if conf['snmpy_global']['create_mib']:
         conf['snmpy_global']['create_mib'].write(snmpy.mibgen.create_mib(conf, mods))
+        conf['snmpy_global']['create_mib'].flush()
```

Comment:
This is implicit on close/exit, no need to manually flush.  However testing this, I've discovered a different problem where the pidfile gets created irrespective of `--create-mib` flag.